libr/anal/esil.c

```diff
@@ -105,17 +105,30 @@ R_API RAnalEsil *r_anal_esil_new(int stacksize, int iotrap, unsigned int addrsiz
 	return esil;
 }
 
-R_API int r_anal_esil_set_op(RAnalEsil *esil, const char *op, RAnalEsilOp code) {
+R_API bool r_anal_esil_set_op(RAnalEsil *esil, const char *op, RAnalEsilOpCb code, ut32 push, ut32 pop, ut32 type) {
 	char t[128];
 	if (!code || !op || !strlen (op) || !esil || !esil->ops) {
 		return false;
 	}
 	char *h = sdb_itoa (sdb_hash (op), t, 16);
-	sdb_num_set (esil->ops, h, (ut64)(size_t)code, 0);
-	if (!sdb_num_exists (esil->ops, h)) {
-		eprintf ("can't set esil-op %s\n", op);
-		return false;
+	RAnalEsilOp *operation = sdb_num_get (esil->ops, h, 0);
```

Comment:
uh? sdb_num_get returns ut64, not void * that should at least warn without any sizet cast

Reply:
can we use "op" instead of "operation" ?

Reply:
ping, fix this

Reply:
op is already used as an argument passed to this function

Reply:
i will use eop